draft-ietf-lamps-x509-shbs.md

```diff
@@ -160,39 +174,53 @@ and discussed later in {{use-cases-shbs-x509}}.
 
 # Use Cases of S-HBS in X.509 {#use-cases-shbs-x509}
 
-As many cryptographic algorithms that are considered to be quantum-resistant,
-S-HBS have several pros and cons regarding their practical usage. On the
-positive side they are considered to be secure against a classical as well as a
-quantum adversary, and a secure instantiation of S-HBS may always be built as
-long as a cryptographically secure hash function exists. Moreover, S-HBS offer
-small public key sizes, and, in comparison to other post-quantum signature
-schemes, the S-HBS can offer relatively small signature sizes (for certain
-parameter sets). While key generation and signature generation may take longer
-than classical alternatives, fast and minimal verification routines can be
-built.  The major negative aspect is the statefulness.  Private keys always
-have to be handled in a secure manner, S-HBS necessitate a special treatment of
-the private key in order to avoid security incidents like signature forgery
-[MCGREW], [SP800208]. Therefore, for S-HBS, a secure environment MUST be used
-for key generation and key management.
-
-Note that, in general, root CAs offer such a secure environment and the number
-of issued signatures (including signed certificates and CRLs) is often moderate
-due to the fact that many root CAs delegate OCSP services or the signing of
-end-entity certificates to other entities (such as subordinate CAs) that use
-stateless signature schemes. Therefore, many root CAs should be able to handle
-the required state management, and S-HBS offer a viable solution.
-
-As the above reasoning for root CAs usually does not apply for subordinate CAs,
-it is NOT RECOMMENDED for subordinate CAs to use S-HBS for issuing end-entity
-certificates. Moreover, S-HBS MUST NOT be used for end-entity certificates.
-
-However, S-HBS MAY be used for code signing certificates, since they are
-suitable and recommended in such non-interactive contexts. For example, see the
-recommendations for software and firmware signing in [CNSA2.0]. Some
+As described in the Security Considerations of {{sec-security}}, it is
+imperative that S-HBS implementations do not reuse OTS signatures. This makes
+S-HBS algorithms inappropriate for general use cases. The exact conditions
+under which S-HBS certificates may be used is left to certificate policies,
+however the intended use of S-HBS as described by [SP800208] can be used as a
+guideline:
```

Comment:
There is a double usage of the word "use". I recommend to write instead: 

The exact conditions under which S-HBS certificates may be used is left to certificate policies. However, the recommended scenarios for deploying S-HBS as described in [SP800208] can be used as a guideline: